src/formio/components/FileField.stories.mdx

```diff
@@ -0,0 +1,124 @@
+import {Canvas, Meta, Props, Story} from '@storybook/addon-docs';
+
+import {FileField} from './FileField';
+import {FormDecorator, FormioComponent} from './story-util';
+import {
+  BASE_URL,
+  ENDPOINT,
+  mockFileUploadError,
+  mockFileUploadPost,
+} from '/src/mocks/FileUploadMocks';
```

Comment:
please use 'absolute' imports relative to the `src` folder: `mocks/FileUploadMocks`

related: I prefer keeping the mocks and components closely together, so in this case `./FileUpload.mocks.js` is probably a better location.

#369 also adds some more general structure (exposes `BASE_URL` in `story-utils/mocks.js`) so let's wait until that is merged and then rebase this branch on that.